lib/src/androidTest/java/com/couchbase/lite/QueryTest.java

```diff
@@ -1232,6 +1252,16 @@ public void testQuantifiedOperatorVariableKeyPath() {
         //TODO
     }
 
+    // https://issues.couchbase.com/browse/CBL-71
+    @Test
+    public void testSelectWithAlias() throws Exception {
```

Comment:
seems like this is redundant like iOS, as the same test is already been done in below test `testSelectAll`